src/containers/Tenant/utils/schemaActions.ts

```diff
@@ -51,22 +52,20 @@ const bindActions = (
         selectQuery: inputQuery(selectQueryTemplate),
         upsertQuery: inputQuery(upsertQueryTemplate),
         copyPath: () => {
-            navigator.clipboard
```

Comment:
`navigator.clipboard` was not always available for embedded version, so sometimes copy didn't work. Replaced it with `copy` from `copy-to-clipboard` lib, that is used in `react-copy-to-clipboard`, that is used in `uikit`, so similar behaviour is expected